.github/workflows/test.yml

```diff
@@ -1,7 +1,8 @@
 name: Tests
-on: [push, pull_request]
+on: pull_request
```

Comment:
Can you keep the builds on push? There are useful when developing a branch for a while

Reply:
done @amercader 
I don't have permission to merge this, so you should.